### DISCUSSION TITLE
有Semi Design Pro 最佳实践计划吗

### DISCUSSION THREAD

**Discussion by UniKylin** [DouyinFE/semi-design#288] [Ideas]
🔼

**Comment by UniKylin** [comment]
1. Semi Design Vue Pro
2. Semi Design React Pro
3. Semi Design Mobile

**Reply by Caesar-APAX** [reply]
你不如直接用antd, 祖宗UI

**Comment by wuhw** [comment]
对于最佳实践，目前还没有准确的时间节点。我们内部这周会就Pro & 物料平台后续的产品和开源规划做些讨论，本月会将结果同步出来

**Comment by xieyezi** [comment]
@UniKylin 
https://github.com/xieyezi/semi-design-pro
